playkit/src/main/java/com/kaltura/playkit/plugins/ads/AdEvent.java

```diff
@@ -253,7 +277,9 @@ public DAISourceSelected(String sourceURL) {
         AD_BUFFER_END,
         AD_PLAYBACK_INFO_UPDATED,
         ERROR,
-        DAI_SOURCE_SELECTED
+        DAI_SOURCE_SELECTED,
+        AD_WATERFALLING,
```

Comment:
@GouravSna  what is this event meaning

Reply:
This is when the AdWaterfalling start then it will be fired and if all the ads failed in the waterfalling list then AD_WATERFALLING_FAILED will be fired.